shavar/e2e-test/prefs.ini

```diff
@@ -3,29 +3,31 @@
 ################################
 
 [DEFAULT]
-browser.startup.homepage = http://itisatrap.org/firefox/its-a-tracker.html 
+browser.startup.homepage = http://itisatrap.org/firefox/its-a-tracker.html
 browser.safebrowsing.debug = true
 browser.safebrowsing.enabled = false
 browser.safebrowsing.malware.enabled = false
 privacy.trackingprotection.enabled = true
 browser.safebrowsing.provider.mozilla.nextupdatetime = 1
-#private.browsing.autostart = true
+browser.safebrowsing.provider.mozilla.gethashURL = ""
+
+# For these prefs, the values need to be ADDED to the existing ones
```

Comment:
doing +=  might be challenging.  normally we launch firefox with these pref sets and the new profile is created at launch time with whatever you specify.  
@fmarier   do you know if firefox can accept this kind of += concatenation when you load custom prefs at launch time?